go/flags/endtoend/vtgate.txt

```diff
@@ -1,5 +1,5 @@
 Usage of vtgate:
-      --allowed_tablet_types []topodatapb.TabletType                     Specifies the tablet types this vtgate is allowed to route queries to.
+      --allowed_tablet_types strings                                     Specifies the tablet types this vtgate is allowed to route queries to. Should be provided as a comma-separated set of tablet types.
```

Comment:
I now see that we are being inconsistent. this is `strings` but `tablet_types_to_wait` is string. One of the two needs to change. We should also review everything that is currently `strings`.

Reply:
I changed tablet_types_to_wait to []topodatapb.TabletType as well.